### DISCUSSION TITLE
刷肉鸽时会卡在“准备作战”界面上

### DISCUSSION THREAD

**Discussion by S1xshot** [MaaAssistantArknights/MaaAssistantArknights#5874] [General]
maa刷肉鸽时，有时会卡在“指定完不同角色的技能，准备作战”这个界面上。
我刷萨米肉鸽时发现了这个问题，然后我改刷傀影肉鸽时问题复现了。
卡住的界面上最后一个角色是红，系统默认选择了二技能，说明给角色分配不同的技能这个功能还是好好完成了的，但“开始行动”这个按钮就不点了。
maa这边的数据记录是这样的
08-08 05:37:14 关卡：不期而遇
08-08 06:14:50 关卡识别错误
08-08 06:15:23 已放弃本次探索

**Comment by v3i1y** [comment]
我也是肉鸽卡住，但是跟楼主不一样，我是卡在开始行动的界面上，最后一条记录是 开始作战：关卡名，一卡就是几个小时，手动点开始作战以后似乎能继续运行，不知道是什么原因。

**Reply by v3i1y** [reply]
换回了老版本4.19.3就没卡了，打的是傀影肉鸽，是不是新版本有什么问题？

**Reply by VitaeL04213** [reply]
同样的情况

**Comment by S1xshot** [comment]
我这次刷萨米肉鸽时又碰到开局卡住了的情况，是卡在“选择支援”界面上。选项一是热水壶，选项二是出发前演讲，选项三是回收战利品。默认选中的是选项一。
